legend-engine-config/legend-engine-connection-integration-tests/src/test/java/org/finos/legend/engine/connection/test/AbstractConnectionFactoryTest.java

```diff
@@ -37,6 +37,7 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+// phase 2 - we should move all the connector stuff into legend-xts-connection ?
```

Comment:
will do. This module though is something I intend to keep around, it's the catch-all module for all connection testings, I figured it's more convenient than later have to collect test results and assemble the matrix table.

We can also distribute it, but let's discuss.